uint8 READY = 0
uint8 LOAD_NEW_ROAD = 1
uint8 REMOVE_OLD_TILES = 2
uint8 RENDER_NEW_TILES = 3

uint8 status
string road
string seed

uint16 processed_tiles
uint16 number_of_tiles
